packages/app/src/cli/services/dev/output.ts

```diff
@@ -76,7 +76,11 @@ export function renderDev(
   const shortcuts = []
   const enabledStorePreviewShortcut = developmentStorePreviewEnabled !== undefined
   if (enabledStorePreviewShortcut) {
-    shortcuts.push(buildDevPreviewShortcut(developmentStorePreviewEnabled, apiKey, token))
+    // Enable dev preview on app dev start
+    await developerPreviewUpdate(apiKey, token, true)
+    buildDevPreviewShortcut(true, apiKey, token)
+
+    shortcuts.push(buildDevPreviewShortcut(true, apiKey, token))
```

Comment:
```ts
const result = await developerPreviewUpdate(apiKey, token, true)
shortcuts.push(buildDevPreviewShortcut(result, apiKey, token))
```

I would add an utility method `enableDeveloperPreview` to avoid using `true` or `false` in cases where we know in advance the action to execute